xbmc/cores/DllLoader/Win32DllLoader.cpp

```diff
@@ -23,83 +20,92 @@
-  { "_findfirst64i32",            -1, (void*)dll_findfirst64i32,            NULL },
-  { "_findnext64i32",             -1, (void*)dll_findnext64i32,             NULL },
-  { "fclose",                     -1, (void*)dll_fclose,                    (void*)track_fclose},
-  { "feof",                       -1, (void*)dll_feof,                      NULL },
-  { "fgets",                      -1, (void*)dll_fgets,                     NULL },
-  { "fopen",                      -1, (void*)dll_fopen,                     (void*)track_fopen},
-  { "fopen_s",                    -1, (void*)dll_fopen_s,                   NULL },
-  { "putc",                       -1, (void*)dll_putc,                      NULL },
-  { "fputc",                      -1, (void*)dll_fputc,                     NULL },
-  { "fputs",                      -1, (void*)dll_fputs,                     NULL },
-  { "fread",                      -1, (void*)dll_fread,                     NULL },
-  { "fseek",                      -1, (void*)dll_fseek,                     NULL },
-  { "ftell",                      -1, (void*)dll_ftell,                     NULL },
-  { "getc",                       -1, (void*)dll_getc,                      NULL },
-  { "fgetc",                      -1, (void*)dll_getc,                      NULL },
-  { "rewind",                     -1, (void*)dll_rewind,                    NULL },
-  { "vfprintf",                   -1, (void*)dll_vfprintf,                  NULL },
-  { "fgetpos",                    -1, (void*)dll_fgetpos,                   NULL },
-  { "fsetpos",                    -1, (void*)dll_fsetpos,                   NULL },
-  { "_stati64",                   -1, (void*)dll_stati64,                   NULL },
-  { "_stat64",                    -1, (void*)dll_stat64,                    NULL },
-  { "_stat64i32",                 -1, (void*)dll_stat64i32,                 NULL },
-  { "_fstati64",                  -1, (void*)dll_fstati64,                  NULL },
-  { "_fstat64",                   -1, (void*)dll_fstat64,                   NULL },
-  { "_fstat64i32",                -1, (void*)dll_fstat64i32,                NULL },
-  { "_telli64",                   -1, (void*)dll_telli64,                   NULL },
-  { "_tell",                      -1, (void*)dll_tell,                      NULL },
-  { "_fileno",                    -1, (void*)dll_fileno,                    NULL },
-  { "ferror",                     -1, (void*)dll_ferror,                    NULL },
-  { "freopen",                    -1, (void*)dll_freopen,                   (void*)track_freopen},
-  { "fscanf",                     -1, (void*)dll_fscanf,                    NULL },
-  { "ungetc",                     -1, (void*)dll_ungetc,                    NULL },
-  { "_fdopen",                    -1, (void*)dll_fdopen,                    NULL },
-  { "clearerr",                   -1, (void*)dll_clearerr,                  NULL },
+  { "_close",                     -1, (void*)dll_close          },
```

Comment:
```suggestion
  { "_close",                     -1UL, (void*)dll_close          },
```

Not directly related with this PR but changing -1 to -1UL fixes a lot of warnings:

```
04:56:31 C:\jenkins-workspace\workspace\WIN-64\xbmc\cores\DllLoader\Win32DllLoader.cpp(39,36): warning C4838: conversion from 'int' to 'unsigned long' requires a narrowing conversion [C:\jenkins-workspace\workspace\WIN-64\kodi-build.x64\libkodi.vcxproj]
04:56:31 C:\jenkins-workspace\workspace\WIN-64\xbmc\cores\DllLoader\Win32DllLoader.cpp(40,36): warning C4838: conversion from 'int' to 'unsigned long' requires a narrowing conversion [C:\jenkins-workspace\workspace\WIN-64\kodi-build.x64\libkodi.vcxproj]
04:56:31 C:\jenkins-workspace\workspace\WIN-64\xbmc\cores\DllLoader\Win32DllLoader.cpp(42,36): warning C4838: conversion from 'int' to 'unsigned long' requires a narrowing conversion [C:\jenkins-workspace\workspace\WIN-64\kodi-build.x64\libkodi.vcxproj]
04:56:31 C:\jenkins-workspace\workspace\WIN-64\xbmc\cores\DllLoader\Win32DllLoader.cpp(43,36): warning C4838: conversion from 'int' to 'unsigned long' requires a narrowing conversion [C:\jenkins-workspace\workspace\WIN-64\kodi-build.x64\libkodi.vcxproj]
04:56:31 C:\jenkins-workspace\workspace\WIN-64\xbmc\cores\DllLoader\Win32DllLoader.cpp(44,36): warning C4838: conversion from 'int' to 'unsigned long' requires a narrowing conversion [C:\jenkins-workspace\workspace\WIN-64\kodi-build.x64\libkodi.vcxproj]
04:56:31 C:\jenkins-workspace\workspace\WIN-64\xbmc\cores\DllLoader\Win32DllLoader.cpp(45,36): warning C4838: conversion from 'int' to 'unsigned long' requires a narrowing conversion [C:\jenkins-workspace\workspace\WIN-64\kodi-build.x64\libkodi.vcxproj]
04:56:31 C:\jenkins-workspace\workspace\WIN-64\xbmc\cores\DllLoader\Win32DllLoader.cpp(46,36): warning C4838: conversion from 'int' to 'unsigned long' requires a narrowing conversion [C:\jenkins-workspace\workspace\WIN-64\kodi-build.x64\libkodi.vcxproj]
04:56:31 C:\jenkins-workspace\workspace\WIN-64\xbmc\cores\DllLoader\Win32DllLoader.cpp(47,36): warning C4838: conversion from 'int' to 'unsigned long' requires a narrowing conversion [C:\jenkins-workspace\workspace\WIN-64\kodi-build.x64\libkodi.vcxproj]
```

Reply:
Yea thanks. I can do some extra adjustments in a follow up PR.